src/utils/textParse.js

```diff
@@ -32,11 +34,18 @@ function parseMentions(text, parameters) {
 	if (Object.keys(parameters).some(key => key.startsWith('mention'))) {
 		for (const [key, value] of Object.entries(parameters)) {
 			let mention = ''
-			if (value?.type === 'call') {
+
+			if (key.startsWith('mention-call') && value?.type === 'call') {
 				mention = '@all'
-			} else if (value?.type === 'user') {
+			} else if (key.startsWith('mention-federated-user') && value?.type === 'user') {
+				const server = value.server ?? getBaseUrl().split('//').pop()
+				mention = `@"federated_user/${value.id}@${server}"`
+			} else if (key.startsWith('mention-group') && value?.type === 'user-group') {
+				mention = `@"group/${value.id}"`
```

Comment:
This can be broken when group name has spaces
![image](https://github.com/nextcloud/spreed/assets/84044328/8a81bcec-60d6-41d7-8335-f4cd3c773a0e)

maybe change the regex to :
```js
text = text.replace(new RegExp(`\\b{${key}}\\b`, 'g'), mention)
```

Reply:
will check later

Reply:
Works correctly:
![image](https://github.com/nextcloud/spreed/assets/93392545/78e55e0b-f7a1-40db-b12f-8283f6c0d756)

Reply:
Only if there is no text after it. Any text that comes after the mention will be included in the quotation.